server/src/test/java/org/spine3/server/entity/storage/EntityQueryShould.java

```diff
@@ -77,12 +80,12 @@ public void support_equality() {
         final Map<Column<?>, Object> paramsCCopy = new HashMap<>(1);
         paramsA.put(otherColumn, someValue);
 
-        final EntityQuery<?> queryA = EntityQuery.of(idsA, paramsA);
-        final EntityQuery<?> queryB = EntityQuery.of(idsB, paramsB);
-        final EntityQuery<?> queryC = EntityQuery.of(idsC, paramsC);
-        final EntityQuery<?> queryD = EntityQuery.of(idsA, paramsC);
-        final EntityQuery<?> queryE = EntityQuery.of(idsB, paramsB);
-        final EntityQuery<?> queryF = EntityQuery.of(idsC, paramsCCopy);
+        final EntityQuery<?> queryA = EntityQuery.of(idsA, fromValues(paramsA));
```

Comment:
Let's categorize the equality sets by naming the `ids..` and `query...` according to what the IDs represent.